tests/Fixer/ControlStructure/ControlStructureContinuationFixerTest.php

```diff
@@ -0,0 +1,258 @@
+                }',
+            '<?php
+                if ($foo) {
+                    foo();
+                } elseif ($bar) {
+                    bar();
+                }',
+            ['keyword_position' => 'next_line'],
+        ];
+
+        yield 'else if (next line)' => [
+            '<?php
+                if ($foo) {
+                    foo();
+                }
+                else if ($bar) {
+                    bar();
+                }',
+            '<?php
+                if ($foo) {
+                    foo();
+                } else if ($bar) {
+                    bar();
+                }',
+            ['keyword_position' => 'next_line'],
+        ];
+
+        yield 'do while (next line)' => [
+            '<?php
+                do {
+                    foo();
+                }
+                while ($foo);',
+            '<?php
+                do {
+                    foo();
+                } while ($foo);',
+            ['keyword_position' => 'next_line'],
+        ];
+
+        yield 'try catch finally (next line)' => [
+            '<?php
+                try {
+                    foo();
+                }
+                catch (Throwable $e) {
+                    bar();
+                }
+                finally {
+                    baz();
+                }',
+            '<?php
+                try {
+                    foo();
+                } catch (Throwable $e) {
+                    bar();
+                } finally {
+                    baz();
+                }',
+            ['keyword_position' => 'next_line'],
+        ];
+
+        yield 'else with comment after closing brace' => [
+            '<?php
+                if ($foo) {
+                    foo();
+                } // comment
+                else {
+                    bar();
+                }',
+        ];
+
+        yield 'elseif with comment after closing brace' => [
+            '<?php
+                if ($foo) {
+                    foo();
+                } // comment
+                elseif ($bar) {
+                    bar();
+                }',
+        ];
+
+        yield 'else if with comment after closing brace' => [
+            '<?php
+                if ($foo) {
+                    foo();
+                } // comment
+                else if ($bar) {
+                    bar();
+                }',
+        ];
+
+        yield 'do while with comment after closing brace' => [
+            '<?php
+                do {
+                    foo();
+                } // comment
+                while (false);',
+        ];
+
+        yield 'try catch finally with comment after closing brace' => [
+            '<?php
+                try {
+                    foo();
+                } // comment
+                catch (Throwable $e) {
+                    bar();
+                } // comment
+                finally {
+                    baz();
+                }',
+        ];
+    }
```

Comment:
Can we have a test for Windows line endings?

Reply:
Done.